docs/source/install.rst

```diff
@@ -4,33 +4,32 @@
 
 Installing the ROSCO tools
 ===========================
-Both the ROSCO controller and ROSCO toolbox should be installed if one wishes to leverage the full ROSCO toolchain. 
-Of course, some users may only wish to download and install only the controller or the toolbox. 
+Depending on what is needed, a user can choose to use just the ROSCO controller or to use both the ROSCO controller and the toolbox.
+Both the controller and the toolbox should be installed if one wishes to leverage the full ROSCO toolchain.
 
-For users wishing to use just the ROSCO toolbox or the ROSCO toolbox and controller, please skip to the section on section :ref:`installing_rt`. 
-Here we will provide the necessary steps to install the ROSCO toolbox and the ROSCO controller simultaneously. 
-For users planning to only download and compile the ROSCO controller, please follow the instructions on :ref:`compiling_rosco`. 
+For users who to use the ROSCO toolbox (with or without the controller), please skip to the section on section :ref:`rosco_toolbox`. 
```

Comment:
For users who **wish**

Reply:
Resolved in d98a58118e5adf594ab7c411641036b55343a6aa